daffodil-lib/src/main/scala/org/apache/daffodil/cookers/Cookers.scala

```diff
@@ -90,20 +90,18 @@ object RepValueCooker extends ListOfStringLiteralNonEmptyNoCharClassEntitiesNoBy
 
 object UpperCaseTokenCooker extends UpperCaseToken
 
-object IntRangeCooker extends Converter[String, Seq[Range[BigInt]]] {
-  protected def convert(input: String, context: ThrowsSDE, forUnparse: Boolean): Seq[Range[BigInt]] = {
-    def run(xs: Seq[String]): Seq[(RangeBound[BigInt], RangeBound[BigInt])] = {
+object IntRangeCooker extends Converter[String, Seq[(JBigInt, JBigInt)]] {
+  protected def convert(input: String, context: ThrowsSDE, forUnparse: Boolean): Seq[(JBigInt, JBigInt)] = {
+    def run(xs: Seq[String]): Seq[(JBigInt, JBigInt)] = {
       xs match {
         case Seq() => Seq()
         case a +: b +: rest => {
-          val a2 = BigInt(a)
-          val b2 = BigInt(b)
-          if (a2.compare(b2) > 0) {
+          val a2 = new JBigInt(a)
```

Comment:
Even though this usage is transient, we're stamping out all use of BigInt in favor of JBigInt. I think that's good, but have you done that comprehensively, or selectively in this change set?

Reply:
Selectively. I didn't have a good way of finding all instances of scala BigInt. Switching over to DataValue uncovered a lot of them, but it is possible that we have more hiding someplace.

Based on some of the recent debugging that I have been doing, it occurs to me that I can use the dissembler to comprehensibly search for scala's BigInt. I'll give that a go and see if it uncovers anything.

Reply:
I can confirm that we still have usages of scala BigInt and BigDecimal in places. The few I looked into turned out to just be unit tests.

It is probably worth cleaning these up as well, but seems out of scope for this PR.